sdk-lib/src/main/java/cash/z/ecc/android/sdk/SdkSynchronizer.kt

```diff
@@ -386,7 +386,7 @@ class SdkSynchronizer private constructor(
 
     suspend fun refreshUtxos() {
```

Comment:
Should we eliminate this method and just keep the one with the explicit account argument?  I think we're trending towards being explicit about requiring the account argument, although it isn't 100% consistent everywhere.

Reply:
Yes, good idea.

Reply:
I agree that it would be a good idea to just remove this method. Unresolving this conversation so that this can be addressed.